apps/desktop/src/lib/demo.ts

```
@@ -0,0 +1,155 @@
+/** Kind discriminator for the public licensed-demo provenance manifest. */
+export const DEMO_PROVENANCE_KIND = "bandscope.licensed-demo" as const;
+
```

ESLint

[error] Missing JSDoc comment.

(jsdoc/require-jsdoc)